InventoryBuilder.py

```diff
@@ -50,43 +50,43 @@ def vrops_list():
 
         @app.route('/<target>/vcenters/<int:iteration>', methods=['GET'])
         def vcenters(target, iteration):
-            return self.iterated_inventory[str(iteration)]['vcenters'].get(target, {})
+            return self.iterated_inventory.get(str(iteration)).get('vcenters').get(target, {})
```

Comment:
Did you face any KeyErrors here?

Reply:
Yes, especially when the inventory pod restarts.